arranger/ingresses.yml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: arranger-server
+spec:
+  tls:
+    - hosts:
+        - arranger-server.qa.clin.ferlab.bio
```

Comment:
These domain names are very environment-specific. 

I would put generic entries here like this (but use "\_\_UNDEFINED\_\_", I started with "kustomizeMe" and then decided that "\_\_UNDEFINED\_\_" is more visible in the final output): https://github.com/Ferlab-Ste-Justine/cqdg-orchestrations/blob/master/keycloak/ingress.yml